### DISCUSSION TITLE
Feature: add Flowbite UI component and provide some examples based on it

### DISCUSSION THREAD

**Discussion by lirantal** [arthelokyo/astrowind#51] [Ideas]
It would be nice if we could have integration with Flowbite UI because then folks will be free to further extend and build on their blog and website with the help of ready-made components rather than working it out themselves with tailwind.

**Comment by prototypa** [comment]
Hi @lirantal 

It is a good idea to open the door to a catalog of _Tailwind CSS_ components that allow you to speed up and customize your website.

What I'm not sure yet is whether to tie us to **Flowbite**. In that case we would have a dependency on a product that must be paid for to be used to its full potential. In addition, there are many other component libraries as powerful as Flowbite: **Tailwind UI**, **daisyUI**, **Mamba UI**, **Headless UI**, **Tailwind Elements**, **Meraki UI**, **Xtend UI**, **HyperUI**, ...

Generally, the use of these components requires the installation of a plugin for _Tailwind CSS_, sometimes adding some custom css, and then going to their respective catalogs and obtaining the necessary widgets (and paying in the cases of payment)

What do you think if for now we explain in this post: https://astrowind.vercel.app/useful-resources-to-create-websites (which now has "Lorem ipsum lorem") how AstroWind can be adapted to the main component libraries for Tailwind CSS. And we could also make an examples folder (which can be deleted if not necessary), with configurations ready to be used by all those libraries.

**Reply by lirantal** [reply]
I don't know the others but it makes sense I guess to pick one that's fully open source. I am however not sure this isn't the case with Flowbite, but granted I didn't look too much into their model. The basic components are there as open source and use an acceptable permissive license (MIT).

**Reply by lirantal** [reply]
Just looked quickly at Tailwind UI and they also have a pricing model. They're not even free at all.
